docs/tutorial.rst

```diff
@@ -618,6 +618,102 @@ Once you are satisfied that the demographic history that you have built
+
+.. code-block:: python
+
+    >>> pop_config = msprime.PopulationConfiguration(sample_size=2, initial_size=1000)
+    >>> mig_rate_change = msprime.MigrationRateChange(time=100, rate=0.05)
+    >>> ts = msprime.simulate(
+                population_configurations=[pop_config, pop_config],
+                length=1000,
+                demographic_events=[mig_rate_change, msprime.CensusEvent(time=5000)],
+                recombination_rate=1e-7,
+                random_seed=141)
+
+The resulting tree sequence has nodes on each tree at the specified census time. These are the nodes with IDs 9, 10, 11, 12 and 13:
+
+.. code-block:: python
+
+    >>> for tree in ts.trees():
+        print(tree.draw(format="unicode"))
+
+      14   
+     ┏━┻━┓ 
+     9  12 
+     ┃   ┃ 
+     5   ┃ 
+    ┏┻┓  ┃ 
+    ┃ ┃  4 
+    ┃ ┃ ┏┻┓
+    1 3 0 2
+
+     14     
+    ┏━┻━┓   
+    10 13   
+    ┃   ┃   
+    ┃   6   
+    ┃  ┏┻━┓ 
+    ┃  ┃  4 
+    ┃  ┃ ┏┻┓
+    3  1 0 2
+
+      14   
+     ┏━┻━┓ 
+    13  11 
+     ┃   ┃ 
+     ┃   7 
+     ┃  ┏┻┓
+     6  ┃ ┃
+    ┏┻┓ ┃ ┃
+    1 0 2 3
+
+      15   
+     ┏━┻━┓ 
+     8  11 
+     ┃   ┃ 
+     ┃   7 
+     ┃  ┏┻┓
+     6  ┃ ┃
+    ┏┻┓ ┃ ┃
+    1 0 2 3
+
+This tells us that the genetic material ancestral to the present day sample was held within 5 haplotypes at time 5000. The node table shows us that three of these haplotypes (nodes 9, 10 and 11) were in population 0 at this time, and two of these haplotypes (nodes 12 and 13) were in population 1 at this time.
+
+.. code-block:: python
+
+    >>> print(ts.tables.nodes)
+    id  flags   population  individual  time    metadata
+    0   1       0   -1  0.00000000000000    
+    1   1       0   -1  0.00000000000000    
+    2   1       1   -1  0.00000000000000    
+    3   1       1   -1  0.00000000000000    
+    4   0       1   -1  2350.08685279051815 
+    5   0       1   -1  3759.20387382847684 
+    6   0       0   -1  4234.97992185234671 
+    7   0       1   -1  4598.83898042243527 
+    8   1048576 0   -1  5000.00000000000000 
+    9   1048576 0   -1  5000.00000000000000 
+    10  1048576 0   -1  5000.00000000000000 
+    11  1048576 0   -1  5000.00000000000000 
+    12  1048576 1   -1  5000.00000000000000 
+    13  1048576 1   -1  5000.00000000000000 
+    14  0       1   -1  5246.90282987397495 
+    15  0       0   -1  8206.73121309170347
+
+If we wish to study these ancestral haplotypes further, we can simplify the tree sequence with respect to the census nodes and perform subsequent analyses on this simplified tree sequence:
+
+.. code-block:: python
+
+    >>> nodes = [i.id for i in list(ts.nodes()) if i.flags==msprime.NODE_IS_CEN_EVENT]
+    >>> ts_anc = ts.simplify(samples=nodes)
+
```

Comment:
Maybe print out this simplified ts, and discuss how what it is? I think the last chunk will be a bit mysterious to many users unless you explain it a little bit.

Reply:
I wasn't able to draw the simplified ts because of [this problem](https://github.com/tskit-dev/tskit/issues/236) but have added a bit of extra explanation.